src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/RequestLoggingServiceBehavior.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Just checking -- this looks like a test enhancement unrelated to the cert feature.  If so, okay for this PR but would prefer we make separate PR's for separate features.  Don't block on this.